src/services/job-processor/service.ts

```diff
@@ -0,0 +1,87 @@
+import type { LoggerService } from 'lido-nanolib'
+import type { ExecutionApiService } from '../execution-api/service.js'
+import type { ConfigService } from '../config/service.js'
+import type { MessagesProcessorService } from '../messages-processor/service.js'
+import type { ConsensusApiService } from '../consensus-api/service.js'
+import type { WebhookProcessorService } from '../webhook-sender/service.js'
+
+type ExitMessage = {
+  message: {
+    epoch: string
+    validator_index: string
+  }
+  signature: string
+}
+
+export type JobProcessorService = ReturnType<typeof makeJobProcessor>
+
+export const makeJobProcessor = ({
+  logger,
+  config,
+  executionApi,
+  consensusApi,
+  messagesProcessor,
+  webhookProcessor,
+}: {
+  logger: LoggerService
+  config: ConfigService
+  executionApi: ExecutionApiService
+  consensusApi: ConsensusApiService
+  messagesProcessor: MessagesProcessorService
+  webhookProcessor: WebhookProcessorService
+}) => {
+  const handleJob = async ({
```

Comment:
Maybe simple `runJob` would be better as it seems that `jobProcessor` is what handles jobs?